interview-questions.md

```diff
@@ -7,52 +7,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What are props in React?
 
-  Your answer:
+  Your answer: Props are used to pull data from the original component.
 
-  Researched answer:
+  Researched answer: props are uni-directional to pass data from the parent and that information can not be changed
 
 
 
 2. What is a DOM event?
 
-  Your answer:
+  Your answer: A DOM event be data the DOM tracks like clicks, hovers, and typing.
 
-  Researched answer:
+  Researched answer: A DOM event lets react track different handlers of the webpage.
 
 
 
 3. What is object-oriented programming? How is it different than functional programming?
 
-  Your answer:
+  Your answer: OOP is using an object style format to write and store data/behavior. The difference between to two is that in OOP data is stored and can be ex/imported.
 
-  Researched answer:
+  Researched answer: OOP is a program that models an object format that can store data, attributes, and behavior. The difference between to two is in a functional programming data can not be stored in objects and can only be manipulated while in a function.
 
 
 
 4. What is the difference between a Float and an Integer in Ruby?
 
-  Your answer:
+  Your answer: A float has decimal places and an integer are whole numbers.
 
-  Researched answer:
+  Researched answer: A float is a number that has decimal places and is a more accurate way to represent numbers and math.
 
 
 
 5. Ruby has an implicit return. What does that mean?
 
-  Your answer:
+  Your answer: Implicit return means typing return is not required and is automatically ran.
 
-  Researched answer:
+  Researched answer: Implicit return means typing return is not required and is automatically ran on the last line of code
```

Comment:
Getting an interview question "correct" and being a compelling interviewee are two different things. These answers will not get you hired. They do not communicate an expertise or an experience with any of these materials. They don't show a curiosity or and understanding of the underlying concepts beyond what can be googled.